### ISSUE TITLE
Race condition error while sharing a track stream with other peers

### ISSUE THREAD

**Issue by HoseinHaqiqian** [flutter-webrtc/flutter-webrtc#1172]
**Describe the bug**
After successfull renegotiation which occurs after sharing one peer audio track by calling `addTrack` method to another peer through the host , i always encounter `race-condition` error after a few seconds. I'm able to hear audio and streams for just a few seconds.

`E/rtc     ( 7829): # `
`E/rtc     ( 7829): # Fatal error in: ../../audio/audio_send_stream.cc, line 393 `
`E/rtc     ( 7829): # last system error: 0 \n`
`E/rtc     ( 7829): # Check failed: !race_checker.RaceDetected()`
`E/rtc     ( 7829): # `
`F/libc    ( 7829): Fatal signal 6 (SIGABRT), code -1 (SI_QUEUE) in tid 7977 (AudioTrackJavaT), pid 7829`

**To Reproduce**

- Create a room and mark one device as host
- try to exchange tracks of two other peers

**Expected behavior**
Succussfully exchange track in the way to hear peers audio on the other peer.

* **Flutter version**: 3.3.4
* **Plugin version**: 
* **OS**: Android - It works perfectly on IOS
* **OS version**: All version

**Comment by HoseinHaqiqian** [comment]
Here is the link of issue in [stackoverflow](https://stackoverflow.com/questions/74444054/flutter-webrtc-race-detected-audio-sending-stream-cc)

**Comment by AhmadRaza159** [comment]
did you find any solution?

**Comment by HoseinHaqiqian** [comment]
@AhmadRaza159  Nope, This is known issue with webrtc-android when you try to implement video call over SFU protocol.
